> > > db.gpkg
> > 
> > 
> > I had a look but no luck:
> > ```shell
> > ale@blackhole ~/dev/QGIS $ grep -rl db.gpkg  tests
> > tests/testdata/qgis_server/test_project_values_represent_attrs.qgs   # untracked! probably a leftover of mine's.
> > ale@blackhole ~/dev/QGIS$ grep -rl represent_attrs  tests
> > [ nothing returned ]
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Looks like it isn't used from any test. I wonder if is there any data item test that scans the folder and opens it.
> 
> I pushed a commit REMOVING the file, to see which test would fail, and:
> 
> https://github.com/qgis/QGIS/runs/6856683069?check_suite_focus=true#step:13:530 OGR(1): Cannot open /root/QGIS/tests/testdata/qgis_server/db.gpkg.()
> 
> https://github.com/qgis/QGIS/runs/6856683069?check_suite_focus=true#step:13:557 CMake Error at PyQgsServerWMSGetFeatureInfoPG.cmake:23 (message): Test failed: 1
> 
> ```
> 803 - PyQgsServerWMSGetFeatureInfoPG (Failed)
> ```

Oh yeah, forgot about qgz. 

Now I can fix it.

--- a/.docker/docker-compose-testing-postgres.yml
+++ b/.docker/docker-compose-testing-postgres.yml
@@ -23,6 +23,5 @@ services:
     environment:
       - LANG=C.UTF-8
       - LC_ALL=en_US.UTF-8
-      - QGIS_TEST_ACCEPT_GITSTATUS_CHECK_FAILURE=1
     cap_add:
       - NET_ADMIN

